staging/gatekeeper/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "3.2.2"
 description: Gatekeeper - Policy Controller for Kubernetes
 name: gatekeeper
-version: 0.5.2
+version: 0.5.3
```

Comment:
of `appVersion` changes a minor or major version, change this to reflect that.

Reply:
ill bump to minor version instead of the patch.